v2/Makefile

```diff
@@ -89,7 +89,9 @@ extract:
 	$(extract_pl) ../v1/cn/index.html cn
 
 .PHONY: gendata
-gendata: $(tsv_files)
+gendata:
+	node ./util/create-blogs-template.js
+	$(tsv_files)
```

Comment:
This looks wrong. Why moving `$(tsv_files)` from the dependency list? You should really learn Makefile syntax and semantics first. If your `create-blogs-template.js` generate files, then those files should be goals of separate Makefile rules.

Reply:
new Makefile rule added.